src/controllers/orbit-controller/orbit-state.js

```diff
@@ -0,0 +1,377 @@
+import OrbitViewport from '../../lib/viewports/orbit-viewport';
+import vec3_add from 'gl-vec3/add';
```

Comment:
I'm still expecting to extract these viewport control components to their own package some day (and share between react-map-gl and deck.gl). The Vector3 class requires luma.gl as a dependency.